README.md

```diff
@@ -51,7 +51,8 @@ federation (2 subschema services implemented via `GraphQLComponent` and a vanill
   - `dataSources` - an array of data sources instances to make available on `context.dataSources` .
   - `dataSourceOverrides` - overrides for data sources in the component tree.
   - `federation` - make this component's schema an Apollo Federated schema (default: `false`).
-  - `makeExecutableSchema` - an optional custom implementation of the function which builds the executable schema from the provided type definitions, imports and resolvers. Defaults to the version provided by the [`graphql-tools`](https://www.graphql-tools.com/docs/generate-schema) package. Not used when `federation` is set to `true`.
+  - `pruneSchema` - prune the schema according to [pruneSchema in graphql-tools](https://www.graphql-tools.com/docs/api/modules/utils_src#pruneschema)
```

Comment:
As both pruneSchema  and pruneSchemaOptions  are optional in GraphQLComponentOptions, we can mention that in the readme, might be more clear to the consumers of the package.